metaci/testresults/templates/testresults/build_flow_tests.html

```diff
@@ -81,47 +81,59 @@
-                <td><a href="{{ result.result.get_absolute_url }}">{{ result.result.method.name }}</a>
-                    <a href="/tests/trend/method/{{ result.result.method.id }}" title="Trend">
-                      <span class="slds-icon_container slds-icon-standard-metric" title="Trend">
-                        {% autoescape off %}
-                        <svg aria-hidden="true" class="slds-icon">
-                          <use xlink:href="/static/design-system/assets/icons/standard-sprite/svg/symbols.svg#metrics"></use>
-                        </svg>
-                        {% endautoescape %}
-                        <span class="slds-assistive-text">Like</span>
-                      </span>
-                    </a>
-                </td>
-                
-                <td class="{% if result.result.outcome == 'Pass' %}success{% else %}danger{% endif %}">{{ result.result.outcome }}</td>
-                <td>{{ result.result.duration }}</td>
-                {% for column in result.columns %}
-                <td class="{{ column.status }}">{{ column.value }}</td>
-                {% endfor %}
-              </tr>
+      <table class="slds-table slds-table--bordered slds-table--cell-buffer">
+        <thead>
+          <tr class="slds-text-title--caps">
+            <th>Method</th>
+            <th>Result</th>
+            <th>Duration</th>
+            {% for column in columns %}
+            <th>{{ column }}</th>
             {% endfor %}
-            <tbody>
-          </table>
-      </div>
-      {% endfor %}
+          </tr>
+        </thead>
+        <tbody>
+        {% for result in class_results.results %}
+          <tr>
+            <td><a href="{{ result.result.get_absolute_url }}">{{ result.result.method.name }}</a>
+                <a href="/tests/trend/method/{{ result.result.method.id }}" title="Trend">
+                  <span class="slds-icon_container slds-icon-standard-metric" title="Trend">
+                    {% autoescape off %}
+                    <svg aria-hidden="true" class="slds-icon">
+                      <use xlink:href="/static/design-system/assets/icons/standard-sprite/svg/symbols.svg#metrics"></use>
+                    </svg>
+                    {% endautoescape %}
+                    <span class="slds-assistive-text">Like</span>
+                  </span>
+                </a>
+            </td>
+            
+            <td class="{% if result.result.outcome == 'Pass' %}success{% else %}danger{% endif %}">{{ result.result.outcome }}</td>
+            <td>{{ result.result.duration }}</td>
+            {% for column in result.columns %}
+            <td class="{{ column.status }}">{{ column.value }}</td>
+            {% endfor %}
+          </tr>
+        {% endfor %}
+        <tbody>
+      </table>
+  </div>
+  {% endfor %}
 
+  {% if assets %}
+  <h3 class="slds-text-heading--small slds-p-top--small">Assets:</h3>
+  <ul class="slds-list_dotted">
+    {% for asset in assets %}
```

Comment:
Will a test always have at least one asset? If not, it might be worth adding some code to add "_No assets available_" or some other message so we know explicitly that there are none, rather than wondering why they aren't showing up.

Reply:
@boakley good idea; added.